### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by corinnemrogers**
This is a first step in updating our documentation procedures.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# AtoM documentation
+
+[AtoM](https://www.accesstomemory.org/) is a web-based, open source application for standards-based archival description and access in a multilingual, multi-repository environment.
+Our target users are archivists, librarians, and anyone working to describe and provide access to archival material.
+
```
Comment (fiver-watson) [inline README.md:15]:
```suggestion

## About the AtoM docs

The AtoM documentation can be found on the Access to Memory website, at: 

* https://www.accesstomemory.org/docs

We version the documentation for each major AtoM release. The docs are written in reStructured Text  ([reST](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html)), and generated using the [Sphinx documentation generator](http://www.sphinx-doc.org/). Learn more about our documentation here: 

* https://wiki.accesstomemory.org/Resources/Documentation/About

```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# AtoM documentation
+
+[AtoM](https://www.accesstomemory.org/) is a web-based, open source application for standards-based archival description and access in a multilingual, multi-repository environment.
```
Comment (fiver-watson) [inline README.md]: Should we list the full project name first? i.e.:

```
Access to Memory ([AtoM](https://www.accesstomemory.org/)) is a ...

```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Our target users are archivists, librarians, and anyone working to describe and provide access to archival material.
```
Comment (fiver-watson) [inline README.md]:
```suggestion
Our target users are archivists, librarians, and anyone working to describe and provide access to archival material. This repository holds the documentation for the AtoM project, including the User, Administrator, and Developer Manuals. 
```

**Review by fiver-watson — COMMENTED** [review]
Hey! 

This is looking good! I've added a few suggestions, but feel free to reject or revise them. I also think we could add the Code of conduct right here, if we want, under another h1 heading. 

One general suggestion: even with the README, don't forget to wrap at 80chars!

File: README.md (hunk 2 above)
Comment (corinnemrogers) [reply README.md]: I wondered about that too - yes I think we should

**Comment by fiver-watson** [comment]
We've added this file manually. Closing this PR now. Thanks!

**Pull request closed without merging**